### ISSUE TITLE
Sandbox for Rippling

### ISSUE THREAD

**Issue by marcosmarxm** [airbytehq/airbyte#27266]
### Topic

source-rippling, sandbox

### Revelant information

To run integration tests for: https://github.com/airbytehq/airbyte/pull/27242

**Comment by igrankova** [comment]
@bmcgilli, @oustynova 
Can it be prioritized according to our roadmap?

**Comment by bmcgilli** [comment]
@marcosmarxm @jrolom is this one from the contributor program? can you just explain how this came to us and what needs to happen now?

**Comment by jrolom** [comment]
Yeah the PR with the source connector is at the top - please help provide the sandbox account so that TSE can run integration test

**Comment by bmcgilli** [comment]
@oustynova if we can prioritize this one this week or next, it'd be great. Test account setup so Marcos can review the new connector. Rippling is a major tool so we will want this source connector in our catalog. Thanks!

**Comment by oustynova** [comment]
@bmcgilli yes, sure. @ycherniaiev will take the task this week

**Comment by ycherniaiev** [comment]
Applied for development package. Once your development package has been reviewed and approved, you will receive your Rippling App Testing Package to test and build the integration in Sandbox.

**Comment by ycherniaiev** [comment]
@marcosmarxm we have no feedback from them

**Comment by ycherniaiev** [comment]
![image](https://github.com/airbytehq/airbyte/assets/94798230/2fb48ee6-1bc1-4584-9db6-e5abb38bd82f)
Seems like no way to obtain sandbox now

**Comment by marcosmarxm** [comment]
@ycherniaiev we sent another email to Rippling to check if possible to allow to creation of the connector. Thanks for all the help.
